test-cases/tests/backup-restore/j08-verify-user-sso-backup-and-restore.md

```diff
@@ -20,21 +20,47 @@ Note: this test should only be performed at a time it will not affect other ongo
 
 ### Postgres
 
-1. Verify Clients and Realms exist in postgres using the terminal in the `standard-auth` pod in the `redhat-rhmi-operator` namespace
+1. Verify Clients and Realms exist in postgres
 
-```
-# password and host retrieved from rhssouser-postgres-rhmi secret in redhat-rhmi-operator, psql will prompt for password
-psql --host=<<db host> --port=5432 --username=postgres --password --dbname=postgres
-$ select * from clients;
-$ select * from realms;
-```
+   Create a throwaway Postgres instance to access the SSO database
 
-3. Run the backup and restore script
+   ```sh
+   cat << EOF | oc create -f - -n redhat-rhmi-operator
+     apiVersion: integreatly.org/v1alpha1
+     kind: Postgres
+     metadata:
+       name: throw-away-postgres
+       labels:
+         productName: productName
+     spec:
+       secretRef:
+         name: throw-away-postgres-sec
+       tier: development
+       type: workshop
+   EOF
+   ```
 
-```sh
-cd test/scripts/backup-restore
-./j08-verify-user-sso-backup-and-restore.sh | tee test-output.txt
-```
+   Open a terminal in the Pod created for the throwaway Postgres and run the following
 
-4. Wait for the script to finish without errors
-5. Verify in the `test-output.txt` log that the test finished successfully.
+   ```sh
+   # password and host retrieved from rhssouser-postgres-rhmi secret in redhat-rhmi-operator, psql will prompt for password
+   psql --host=<<db host> --port=5432 --username=postgres --password --dbname=postgres
+   $ select * from clients;
```

Comment:
@sergioifg94 as noted by @briangallagher on review of rhsso backup and restore automation. It should be `client` not `clients`.